src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -620,8 +620,7 @@ public static function cacheAndIdentifyUploadedTrack($tmp_path)
         $getID3 = new \getID3();
         $fileInfo = $getID3->analyze(Config::$audio_upload_tmp_dir.'/'.$filename);
         $getID3_lib = new \getID3_lib();
-        $getID3_lib->CopyTagsToComments($fileInfo);
-        
+        $getID3_lib->CopyTagsToComments($fileInfo);       
```

Comment:
You should probably set your editor up to remove trailing whitespace, and to add newlines on the ends of files